lib/hook-template.js

```diff
@@ -1,9 +1,8 @@
 #!/usr/bin/env node
 
-try {
```

Comment:
1. Install hooks.
2. Remove `node_modules` directory.
-> you cannot make a commit ;)

`try..catch` saves the day here.